### PR TITLE
Implements pullup/pulldown resistors configuration in CPU MSP430

### PR DESCRIPTION
**Pull request by Joelfp**

### CODE REVIEW

File: cpu/msp430fxyz/include/msp430_regs.h [hunk 1]
```diff
@@ -57,6 +57,7 @@ typedef struct {
     REG8    OD;         /**< output data */
     REG8    DIR;        /**< pin direction */
     REG8    SEL;        /**< alternative function select */
+    REG8	REN;		/**< eneble pullup/pulldonw resistor */
```
Comment (OlegHahm) [inline cpu/msp430fxyz/include/msp430_regs.h]: Please don't use tabs.

**Comment by Joelfp** [comment]
Why Travis Complains?

**Comment by authmillenon** [comment]
1) there are still trailing whitespaces in your commit:
```diff
+    if (port == NULL) 
cpu/msp430fxyz/periph/gpio.c:85: trailing whitespace.
+    if (port == NULL) 
```

2) a maintainer needs to approve the build by setting the "Ready for CI build" label (this was established to keep the build queues short to prevent PRs to wait for hours or even days to be completed). This will usually happens as soon as the maintainer approves the PR for merging (AKA when she gives her ACK).

File: cpu/msp430fxyz/include/msp430_regs.h [hunk 2]
```diff
@@ -57,6 +57,7 @@ typedef struct {
     REG8    OD;         /**< output data */
     REG8    DIR;        /**< pin direction */
     REG8    SEL;        /**< alternative function select */
+    REG8    REN;        /**< eneble pullup/pulldonw resistor */
```
Comment (authmillenon) [inline cpu/msp430fxyz/include/msp430_regs.h]: enable, pulldown

File: cpu/msp430fxyz/include/msp430_regs.h [hunk 3]
```diff
@@ -70,6 +71,7 @@ typedef struct {
     REG8    IES;        /**< interrupt edge select */
     REG8    IE;         /**< interrupt enable */
     REG8    SEL;        /**< alternative function select */
+    REG8    REN;        /**< eneble pullup/pulldonw resistor */
```
Comment (authmillenon) [inline cpu/msp430fxyz/include/msp430_regs.h]: enable, pulldown

**Comment by Joelfp** [comment]
I am closing this PR since I hadn't read the guidelines. This PR comes from master branch. I will create a branch and make a PR again.

**Pull request closed without merging**